docs/reference/inputs.rst

```diff
@@ -13,6 +13,12 @@ Inputs
 
 -   :class:`web_poet.BrowserResponse`
 
+-   :class:`web_poet.AnyResponse`
+
+    This re-uses either :class:`web_poet.BrowserResponse` *(takes priority)*
+    or :class:`web_poet.HttpResponse` if they're available. If neither is
+    available, then it won't cause either one to be obtained from Zyte API.
```

Comment:
Hm, this doesn't look like a desired behavior. I think if neither is available, we should get one of them.

Reply:
Would `BrowserResponse` be preferred to be obtained if neither is available?

Reply:
I think it should be HttpResponse if neither is available.

Reply:
Thanks for confirming. Updated it in https://github.com/scrapy-plugins/scrapy-zyte-api/pull/161/commits/137d146b13d78a8c55253955183f0509f0d9fc76